test/test_query.cpp

```diff
@@ -4107,6 +4107,46 @@ TEST(Query_LinkChainSortErrors)
     CHECK_LOGIC_ERROR(t1->get_sorted_view(SortDescriptor({{t1_linklist_col}})), ErrorCodes::InvalidSortDescriptor);
 }
 
+TEST_TYPES(Query_SortingCompressedStrings, std::true_type, std::false_type)
```

Comment:
Its a pity we can't use our existing sorting tests just because they don't use transactions. Is there something easy we could do so that our existing sorting tests could cover both the with and without compression cases?

Reply:
Not really (unless I don't rework all the tests to commit first). Essentially, we need to commit in order to compress stuff.